test/integration/config/server-memory.json

```diff
@@ -39,9 +41,24 @@
         {
           "RecordObject:_record_key": "factory",
           "RecordObject:_record_value": { "@id": "urn:solid-server:default:ServerFactory" }
+        },
+        {
+          "comment": "Timer needs to be stopped when tests are finished.",
+          "RecordObject:_record_key": "expiringStorage",
+          "RecordObject:_record_value": { "@id": "urn:solid-server:default:ExpiringIdpStorage" }
         }
       ]
     },
+    {
+      "comment": "Use an actual e-mail sender to mock in the integration test.",
```

Comment:
```suggestion
```

Reply:
Wondering what exactly the issue is with having comments like this? I actually tried to add some more this time so it's more clear why some things are in the config.

Reply:
I don't understand the comment actually. Does it mean:
- We're using an actual email sender here.
- You need to replace this with an actual email sender to make the test work.
- This works, but not in the integration test, where you need to mock it.